extensions/intellij/intellij_generator_plugin/src/main/resources/templates/rx_bloc_with_default_states/rx_bloc.dart.template

```diff
@@ -12,19 +12,19 @@ abstract class ${bloc_pascal_case}BlocEvents {
 /// A contract class containing all states of the ${bloc_pascal_case}BloC.
 abstract class ${bloc_pascal_case}BlocStates {
   /// The loading state
-  Stream<bool> get isLoading;
+  Stream<bool>${bloc_use_null_safety} get isLoading;
 
   /// The error state
-  Stream<String> get errors;
+  Stream<String>${bloc_use_null_safety} get errors;
 }
 
 @RxBloc()
 class ${bloc_pascal_case}Bloc extends ${bloc_dollar_pascal_case}Bloc {
   /// TODO: Implement error event-to-state logic
   @override
-  Stream<String> _mapToErrorsState() =>
+  Stream<String>${bloc_use_null_safety} _mapToErrorsState() =>
```

Comment:
@nikolayIvanovPrime, could you please explain why the mapper needs to be nullable?